locales/de-DE/translation.json

```
@@ -0,0 +1,250 @@
+{
+    "$Language": {
+        "en": "English",
+        "pt-BR": "Português - Brasil",
+        "de-DE": "Deutsch"
```

There are two languages missing here:

```suggestion
        "es": "Español",
        "it": "Italiano",
        "de-DE": "Deutsch"
```